### DISCUSSION TITLE
osu!mania arrow colors

### DISCUSSION THREAD

**Discussion by githubforzack** [ppy/osu#18538] [Q&A] [answered]
i imported my skins from stable to lazer and now my arrow colours are gone?? it's pure white and I want the original green of my skin back how do I fix this?? or is this a bug 
also same with the "judgements"
like the 300's, 200 (100k), 100, etc etc are the default osu skin even though my mania skin already has judgements?? none for 300+, 300, 100k, 100, 50, X for miss.

why does lazer overwrite these files??

![hit100k@2x](https://user-images.githubusercontent.com/88392291/171773666-5f0d95d2-d109-4771-a68c-3ab3a6e53e50.png)

pretend this arrow except its fully white
![down](https://user-images.githubusercontent.com/88392291/171773868-c78c7639-ea47-4b34-b4c1-1058633d4929.png)

edit: skin attached
https://drive.google.com/drive/folders/19EV6bxxbXsGl8eX2USP93LNWwIrX4Eot?usp=sharing

**Comment by frenzibyte** [comment]
Please attach your skin.

**Reply by githubforzack** [reply]
uhh it wonty let me because it doesn't support that type of file ill try putting it in a google drive give me a sec

edit I think I broke google drive

edit 2 i think i may have accidentally uploaded the entirety of osu onto my drive

edit 3 I am currently uploading it to my drive will be done in about 3 mins

**Reply by githubforzack** [reply]
hi my skin is on the main post now

**Comment by smoogipoo** [comment, marked as answer]
This is because lazer always uses the `@2x` files if they exist. And this is how they appear in your skin:
![image](https://user-images.githubusercontent.com/1329837/171783467-d922352f-bb29-4916-8a11-693952d3892f.png)

You'll get the same behaviour in stable as in lazer if you used a widescreen resolution (I think at least 1366x768 if I remember correctly).
